scripts/install-cni.sh

```diff
@@ -121,6 +121,20 @@ else
   cni_spec=${cni_spec//@cniCiliumPlugin/}
 fi
 
+## Look for optional env var to know if plug-in should be enabled
+if [[ -n "${ISTIO_CNI_CONFIG:-}" ]]; then
+  ENABLE_ISTIO_PLUGIN=true
+fi
+
+# Add plug-in to config
+if [ "${ENABLE_ISTIO_PLUGIN}" == "true" ]; then
+  echo "Adding Istio plug-in to the CNI config."
+  cni_spec=${cni_spec//@cniIstioPlugin/, ${ISTIO_CNI_CONFIG}}
```

Comment:
Add @cniIstioPlugin to netd-config in (the sample) netd.yaml.

Reply:
Added @cniIstioPlugin.